src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -124,5 +124,24 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
 
         assertEquals(1, model.getFilteredPersonList().size());
     }
+    /**
+     * Updates {@code model}'s filtered list to show only the persons at the given {@code targetIndices} in the
+     * {@code model}'s address book.
+     */
+    public static void showPersonsAtIndices(Model model, Index[] targetIndices) {
+        assertTrue(targetIndices.length > 0);
+
+        List<String> keywords = new ArrayList<>();
+        for (Index targetIndex : targetIndices) {
+            assertTrue(targetIndex.getZeroBased() < model.getFilteredPersonList().size());
+            Person person = model.getFilteredPersonList().get(targetIndex.getZeroBased());
+            final String[] splitName = person.getName().fullName.split("\\s+");
+            keywords.add(splitName[0]);
+        }
+        model.updateFilteredPersonList(new NameContainsKeywordsPredicate(keywords));
```

Comment:
note: this seems to assume that the first word in a `Person`'s `fullName` is unique in our tests.

this assumption is also done in AB3 so this is ok